sync.go

```diff
@@ -55,10 +55,33 @@ type Group struct {
 	Keys []string `json:"keys"`
 }
 
+func decodeMemberList(body io.ReadCloser) (GoogleMemberList, error) {
+	var memList GoogleMemberList
+
+	dec := json.NewDecoder(body)
+	err := dec.Decode(&memList)
+
+	return memList, err
+}
+
+func (group *Group) addSSHKeys(body io.ReadCloser, m GoogleMember) {
+	var adminUser GoogleAdminUser
+
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(body)
+	b := buf.Bytes()
+	json.Unmarshal(b, &adminUser)
+
+	if len(adminUser.CustomSchemas.Keys.SSH) > 0 {
+		group.Keys = append(group.Keys, adminUser.CustomSchemas.Keys.SSH)
+	} else {
+		log.Printf("Ignoring empty key for member: %s group: %s", m.Email, group.Name)
```

Comment:
Thinking about it, I don't see this log message being valuable, its fairly obvious behaviour of what happens if your key isn't set. So we can probably lose then `else` bit, what do you think?

Reply:
Makes sense, that would actually remove the useless `member` argument from the function as well. It only serves the case where you don't see your key being populated under /authmap and you want to trace it via logs, which is pointless for the case of an empty key. Removing..

Reply:
Ta ^^